### ISSUE TITLE
Drag target reacting to drag position

### ISSUE THREAD

**Issue by VerTiGoEtrex** [flutter/flutter#22857]
Hey, I'm trying to recreate [this effect](https://dribbble.com/shots/4460706-Drag-Drop-upload-concept) I found on Dribbble. Note that as the item is dragged around, a blob follows the pointer until release, and then particles flow from the object to the target. It seems the particles could be done via the current API callbacks since it passes the offset of the DragTarget on drag finish (although this effect would not be straightforward to implement, and I'm OK with ignoring it for now.)

The blob effect, however, seems impossible at the moment. The API for DragTarget does not allow you to react to the position of the DragAnchor _as a drag occurs_ inside (or outside) of its bounds. Any ideas on how you might be able to achieve this type of reaction to a drag based on its current position within the target?

In short, we can react to the position of a drag when it finishes, but not while it's occurring. Is my understanding correct, and if so, are there any quick wins in terms of workarounds?

![effect](https://cdn.dribbble.com/users/149817/screenshots/4460706/comp-3_4.gif)

**Comment by iapicca** [comment]
Hi @VerTiGoEtrex  
This platform is not meant for assistance on personal code.
Please see https://flutter.dev/community for resources and asking questions like this,
you may also get some help if you post it on Stack Overflow.
Closing, as this isn't an issue with Flutter itself,
if you disagree please write in the comments and I will reopen it
Thank you
